EmpleoDotNet/Models/UnitOfWork/UnitOfWork.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
```

Comment:
El pull request solo debería tener el código de social sharing, no de unit of work